src/test/kotlin/org/rustSlowTests/lang/resolve/CargoProjectResolveTest.kt

```diff
@@ -947,6 +947,79 @@ class CargoProjectResolveTest : RsWithToolchainTestBase() {
         prj.checkReferenceIsResolved<RsPath>("project_2/src/main.rs")
     }
 
+    fun `test cargo features namespaced dependencies and weak dependency features`() = buildProject {
+        toml("Cargo.toml", """
+            [package]
+            name = "hello"
+            version = "0.1.0"
+
+            [features]
+            feature_foo = ["dep:foo"]
+            feature_bar = ["dep:bar"]
```

Comment:
[minor] `bar` feature/package seems redundant in this test because it does not check its state